.goreleaser.yaml

```diff
@@ -0,0 +1,29 @@
+before:
+  hooks:
+    - go mod tidy
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    binary: thema
+    main: ./cmd/thema
+
+archives:
+  - format: tar.gz
+    format_overrides:
+    - goos: windows
```

Comment:
For Windows, we'll also probably need to add `.exe` extension to the binary file.

Reply:
I didn't add anything because I wasn't really sure. This is from the basic template.

Reply:
Sure, np! I experienced that in a different project, that's why I'm suggesting it hahaha, but my first experience was exactly as yours (following the basic template), so don't worry at all!

Reply:
I'm seeing into `goreleaser` documentation that available formats are the following:
```
Valid options are `tar.gz`, `tgz`, `tar.xz`, `txz`, tar`, `gz`, `zip` and `binary`.
```

So I understand that could we use `binary`?

Reply:
I think the current format (`zip`) is fine, because it will generate a compressed file with the resulting binary inside (similar to other platforms, just replacing tar by zip, which is way more common on Windows) - that's ok!

What I suggested at the beginning is changing the name of the resulting binary (what will end up within that zip file), from just `thema` (or whatever it is now) to `thema.exe`, because I think that, based on previous experiences, on Windows (by default), a file needs to have the `.exe` extension to be considered an executable 😅 

Hope it makes more sense now! 🙏🏻